w3d_translator/validate.py

```diff
@@ -1,7 +1,10 @@
 import shutil
+from lxml import etree
 from pathlib import Path
 
-from errors import ValidationError
+from errors import ValidationError, XmlError
+
+schema = etree.XMLSchema(file="schema/caveschema.xsd")
```

Comment:
Is this a global variable that doesn't change after this initialization? If so, I'd probably would make it all caps. `SCHEMA`

Reply:
https://peps.python.org/pep-0008/#constants

Reply:
Oh, yes. Good catch! I changed it